Makefile

```diff
@@ -122,6 +122,7 @@ DISABLE_STANDARD_WEBUSB = 1
 DEFINES   += CUSTOM_IO_APDU_BUFFER_SIZE=\(255+5+64\)
 DEFINES   += HAVE_USB_CLASS_CCID
 DEFINES   += HAVE_RSA
+# DEFINES   += WITH_SUPPORT_RSA4096
```

Comment:
Can you add a comment explaining why it's not enabled for now?

Reply:
Done